go-dlv.el

```diff
@@ -38,10 +38,10 @@
 
 (require 'gud)
 
-;; Last group is for return value, e.g. "> test.py(2)foo()->None"
-;; Either file or function name may be omitted: "> <string>(0)?()"
+;; Sample marker line:
+;; > main.main() ./test.go:10 (hits goroutine(5):1 total:1)
 (defvar go-dlv-marker-regexp
-  "^> .+(.*) \\(.+\\)\\:\\([0-9]+\\)$")
+  "^> .+(.*) \\(.+\\)\\:\\([0-9]+\\)\\( (.*)\\)?$")
```

Comment:
`"^> .+(.*) \\(.+\\)\\:\\([0-9]+\\)"` will work as well and simpler and still backward compatible. I will update this PR.